swarm/runtime/_claude_sdk/policy.py

```
@@ -0,0 +1,132 @@
+    Returns:
+        Tuple of (is_blocked, matched_pattern) where matched_pattern is the
+        pattern that matched if blocked, None otherwise.
+    """
+    for pattern in _get_blocked_patterns():
+        if pattern.search(command):
+            return True, pattern.pattern
+    return False, None
+
+
+def create_tool_policy_hook(
+    allow_write: bool = True,
```

Ruff (0.14.11)

Boolean-typed positional argument in function definition

(FBT001)